db/migrate/20200723033409_reindex_post_seach_data.rb

```diff
@@ -0,0 +1,11 @@
+class ReindexPostSeachData < ActiveRecord::Migration[6.0]
+  def up
+    execute <<~SQL
+    UPDATE post_search_data SET version = 0
```

Comment:
The reindex here is intentional so that we update search data for changes like 

https://github.com/discourse/discourse/commit/2196d0b9ae660bab7991608b37d267beb1c16916
https://github.com/discourse/discourse/commit/5c230266d35f07ebadd758f077bc1f2ab5b80b82